lib/cdo/contact_rollups/v2/contact_rollups.rb

```diff
@@ -0,0 +1,6 @@
+class ContactRollupsV2
+  def self.build_contact_rollups
+    ContactRollupsProcessed.delete_all
```

Comment:
should this be a transaction?

Reply:
Good point. This particular table is an intermediate result and will be recreate every time (it doesn't have any use outside of contact rollups pipeline). However, I will think about what is our general rollback strategy for Contact Rollups.

Reply:
I kind of like the idea of treating each step as a "transaction" (ie, it fully completes or it fully fails), then treat contact rollups as a whole as something that could fail part way through, but would only require rerun of steps that either a) never got run, or b) the step where it failed.

Reply:
The ability to recover midpoint in the pipeline is a great thing to have. Let's add that once we have all the pieces connected. For now, I think we can focus on core logic of those pieces first. Thanks Molly for raising this idea.